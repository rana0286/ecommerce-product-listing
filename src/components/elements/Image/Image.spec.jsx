import React from 'react';
import { shallow } from 'enzyme';
import Image from './Image';
import renderer from 'react-test-renderer';

describe('src/components/elements/Image', () => {
  it('Image component should render', () => {
    const component = shallow(<Image src="/testPath" alt="image default" />);
    const tree = renderer.
      create(<Image src="/testPath" alt="image default" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
    expect(component).toBeDefined();
  });
});
