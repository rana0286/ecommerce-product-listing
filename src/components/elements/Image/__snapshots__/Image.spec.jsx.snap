// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/elements/Header Header component should render 1`] = `
<img
  alt="image default"
  className="image"
  src="/testPath"
/>
`;
